src/diffpy/utils/tools.py

```diff
@@ -132,22 +116,50 @@ def get_user_info(owner_name=None, owner_email=None, owner_orcid=None):
             del runtime_info[key]
     global_config = load_config(Path().home() / "diffpyconfig.json")
     local_config = load_config(Path().cwd() / "diffpyconfig.json")
-    # if global_config is None and local_config is None:
-    #     print(
-    #         "No global configuration file was found containing "
-    #         "information about the user to associate with the data.\n"
-    #         "By following the prompts below you can add your name and email to this file on the current "
-    #         "computer and your name will be automatically associated with subsequent diffpy data by default.\n"
-    #         "This is not recommended on a shared or public computer. "
-    #         "You will only have to do that once.\n"
-    #         "For more information, please refer to www.diffpy.org/diffpy.utils/examples/toolsexample.html"
-    #     )
     user_info = global_config
     user_info.update(local_config)
```

Comment:
Since user can only create/update global config, I wonder if it is better to prioritize global to local config?

Reply:
@yucongalicechen thanks for the comment.  User updates local config by placing a file in their local directory.  This is a pretty standard protocol....local overrides global.

Reply:
Gotcha. Then this PR looks great to me. I like the separated structure a lot.

Reply:
Thanks @yucongalicechen !  Yes, it is something I learned at some point.  When you are struggling with function logic it is often because the function is trying to do more than one thing and can be simplified by making two functions that each do one thing.